lms/djangoapps/certificates/__init__.py

```diff
@@ -1,3 +0,0 @@
-""" Certificates app """
-# this is here to support registering the signals in signals.py
-from . import signals
```

Comment:
@yro using `AppConfig` is fine, though why is the import path you're using there different that the one here?